src/pal/pal_net_posix.c

```diff
@@ -1129,6 +1129,9 @@ int32_t pal_ntop(
     prx_size_t addr_string_size
 )
 {
+#ifndef NI_MAXSERV
+#  define NI_MAXSERV 32 // use same definition as in os_mock.h if not defined
```

Comment:
We could also add this to os_posix.h, I wonder though why it is not defined, seems like a standard macro in socket programming...  Maybe I am missing a header.  Did you troll your usr/include to see if it is defined somewhere else or maybe I am missing a __use macro or so... :-(.  Anyway, I will try to repro...